### PR TITLE
Update VideoTrackList for new-style event pages

### PR DESCRIPTION
**Pull request by Elchi3**
This PR updates `VideoTrackList` for new-style event pages. (see the [template](https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_event_subpage_template))

To deduplicate the pages, I used these commands:

```
yarn content delete Web/API/VideoTrackList/onaddtrack --redirect=/en-US/docs/Web/API/VideoTrackList/addtrack_event
yarn content delete Web/API/VideoTrackList/onremovetrack --redirect=/en-US/docs/Web/API/VideoTrackList/removetrack_event
yarn content delete Web/API/VideoTrackList/onchange --redirect=/en-US/docs/Web/API/VideoTrackList/change_event  
```

BCD is approved in https://github.com/mdn/browser-compat-data/pull/13885

### CODE REVIEW

File: files/en-us/web/api/videotracklist/removetrack_event/index.md [hunk 1]
```diff
@@ -5,43 +5,38 @@ tags:
...
-  </tbody>
-</table>
+The `removetrack` event is fired when a video track is removed from a [`VideoTrackList`](/en-US/docs/Web/API/VideoTrackList).
+
+## Syntax
+
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
+
+```js
+addEventListener('removetrack', event => { })
```
Comment (teoli2003) [inline files/en-us/web/api/videotracklist/removetrack_event/index.md:23]: I have a doubt here, shouldn't this be:
```suggestion
addEventListener('removetrack', (event) => { })
```

File: files/en-us/web/api/videotracklist/removetrack_event/index.md [hunk 2]
```diff
@@ -5,43 +5,38 @@ tags:
...
+An {{domxref("TrackEvent")}}. Inherits from {{domxref("Event")}}.
+
+{{InheritanceDiagram("TrackEvent")}}
+
+## Event properties
+
+_In addition to the properties listed below, properties from the parent interface, {{domxref("Event")}}, are available._
+
+- {{domxref("TrackEvent.track", "track")}} {{ReadOnlyInline}}
+  - : The newly-removed {{domxref("VideoTrack")}} the event is in reference to or `null`.
```
Comment (teoli2003) [inline files/en-us/web/api/videotracklist/removetrack_event/index.md:39]: What does `null` means? No track got removed?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Two open questions, else it looks good.

File: files/en-us/web/api/videotracklist/removetrack_event/index.md (hunk 1 above)
Comment (Elchi3) [reply files/en-us/web/api/videotracklist/removetrack_event/index.md:23]: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Functions#arrow_functions says
"If the function only takes one parameter, you can also omit the brackets around the parameter:".

We could have no parameter, but I think that looks too obscure: `addEventListener('removetrack', () => { })`

File: files/en-us/web/api/videotracklist/removetrack_event/index.md (hunk 2 above)
Comment (Elchi3) [reply files/en-us/web/api/videotracklist/removetrack_event/index.md:39]: Hm, I have this from https://developer.mozilla.org/en-US/docs/Web/API/TrackEvent#properties but I think `null` makes no sense. The event is not fired if no track is removed, so `track` shouldn't be `null`. I can remove this, it doesn't add anything.

File: files/en-us/web/api/videotracklist/removetrack_event/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/videotracklist/removetrack_event/index.md:23]: Ha! I didn't know this!  Thanks.

File: files/en-us/web/api/videotracklist/removetrack_event/index.md (hunk 2 above)
Comment (teoli2003) [reply files/en-us/web/api/videotracklist/removetrack_event/index.md:39]: I double-checked. `TrackEvent` is correct in the sense where `null` is possible (in the sense to be _legal_).

But – I checked the HTML spec – in all cases where `addtrack` or `removetrack` events are fired, the `track` field is set to a non-null value. 

So we should remove it here and in `addtrack_event` but not in `TrackEvent`.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
The "or null" to be removed twice and we are good to go.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**